src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1086,6 +1087,60 @@ def load_module(name, value):
     def name_or_path(self) -> str:
         return getattr(self.config, "_name_or_path", None)
 
+    @property
+    def _execution_device(self):
+        r"""
+        Returns the device on which the pipeline's models will be executed. After calling
+        `pipeline.enable_sequential_cpu_offload()` the execution device can only be inferred from Accelerate's module
```

Comment:
Can we format `pipeline.enable_sequential_cpu_offload()` such that the hyperlink generates properly?